pkg/git/git.go

```diff
@@ -1,13 +1,17 @@
 package git
 
 import (
+	"errors"
 	"os"
 	"path/filepath"
 	"strings"
 
+	"github.com/buildsafedev/bsf/pkg/langdetect"
 	"github.com/go-git/go-git/v5"
 )
 
+var ErrFilesNotAddedToVersionControl = errors.New("files like 'package-lock.json', 'Cargo.lock', 'go.mod', 'poetry.lock' are not added to version control")
```

Comment:
Is it possible for us to mention exactly which file hasn't been added rather than a generic message? 🤔

Reply:
It is possible but there are multiple ways. One is redundant, creating 4 errors, each for every language and other can be creating an error type (struct extending error). I feel a need of managing some errors separately by creating a seperate type for them!

Reply:
Maybe we can have a small function that returns this error. The error string can have a `%s` in it so we can change based on context.

Reply:
Fixed!!